pycaret/tests/test_time_series.py

```diff
@@ -470,6 +470,23 @@ def test_tune_model_custom_folds(load_data):
     assert len(metrics2) == custom_fold + 2  # + 2 for Mean and SD
 
 
+def test_tune_model_alternate_metric(load_data):
+    """tests model selection using non default metric"""
+    exp = TimeSeriesExperiment()
+    fh = 12
+    fold = 2
+
+    exp.setup(data=load_data, fold=fold, fh=fh, fold_strategy="sliding")
+
+    model_obj = exp.create_model("naive")
+    tuned_model_obj = exp.tune_model(model_obj, optimize="MAE")
```

Comment:
we could parametrize this to accept `MAE, SMAPE` and all the metrics available in time series module.

Reply:
Yes, good idea. Lets add it as an enhancement to make when we refactor the tests.